microsoft/testsuites/nvme/performance.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from enum import Enum
+from typing import List
+
+from assertpy import assert_that
+
+from lisa import (
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+    simple_requirement,
+)
+from lisa.environment import Environment
+from lisa.features import Nvme, NvmeSettings
+from lisa.notifier import FIOResult, FIOResultsMessage
+from lisa.tools import Echo, Fio, Lscpu
+
+FIOMODES = Enum(
+    "FIOMODES",
+    [
+        "randread",
+        "randwrite",
+        "read",
+        "write",
+    ],
+)
+
+
+@TestSuiteMetadata(
+    area="nvme",
+    category="performance",
+    description="""
+    This test suite is to validate NVMe disk performance of Linux VM using fio tool.
+    """,
+)
+class nvmeperf(TestSuite):  # noqa
+    TIME_OUT = 3000
+
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test NVMe disk performance.
+        """,
+        priority=2,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            supported_features=[NvmeSettings(disk_count=8)],
+        ),
+    )
+    def verify_nvme_performance(self, node: Node, environment: Environment) -> None:
```

Comment:
The case is not to verify something. So, let's use a pattern for perf test like "perf_nvme".

Reply:
updated